### PR TITLE
Added support for Ghostwriter

### PR DESCRIPTION
**Pull request by orschiro**
Dear all, 

Would you mind supporting [Ghostwriter](https://wereturtle.github.io/ghostwriter/)?

Thankfully,

-Robert

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +240,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [Gear Player](https://www.gearmusicplayer.com/)
 - [GeekTool](http://projects.tynsoe.org/en/geektool/)
 - [GHCi](https://wiki.haskell.org/GHC/GHCi)
+  - [Ghostwriter](https://wereturtle.github.io/ghostwriter/)
```
Comment (lra) [inline README.md]: Badly aligned

File: mackup/applications/ghostwriter.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = Ghostwriter
+
+[configuration_files]
+
```
Comment (lra) [inline mackup/applications/ghostwriter.cfg]: This newline is not needed

**Comment by lra** [comment]
just update the same branch.
I suggest some reads https://help.github.com/articles/about-pull-requests/

**Comment by orschiro** [comment]
I think I am getting there to understand how this pull requesting works. :-)

https://github.com/lra/mackup/pull/921

**Pull request closed without merging**